integration/noderunner/noderunner_test.go

```diff
@@ -55,6 +57,8 @@ func TestCanStartStandaloneObscuroHostAndEnclave(t *testing.T) {
 
 	enclaveConfig := config.DefaultEnclaveConfig()
 	enclaveConfig.Address = enclaveAddr
+	empty := common.BytesToAddress([]byte("AA"))
```

Comment:
Can we rename this to `dummyContractAddress` or similar? Think it's clearer than `empty` (assuming that's what these are, I'm not clear).